src/pages/patientView/PatientViewPage.tsx

```diff
@@ -38,10 +38,16 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
             );
         }
 
+        let samples = ["string", "TCGA-24-2024-01", "TCGA-24-2024-01 (deliberately elongated id)"];
```

Comment:
Yes but what we need to pass in is the order of the samples for display, to make it sync'd with the genomic overview.